lib/Phpactor.php

```diff
@@ -94,19 +97,42 @@ public static function boot(InputInterface $input, OutputInterface $output, stri
             $projectRoot = $input->getParameterOption([ '--working-dir', '-d' ]);
         }
 
+        if (!is_string($projectRoot)) {
+            throw new RuntimeException(sprintf(
+                'Unexpected type for project root, expected string got: %s',
+                get_debug_type($projectRoot)
+            ));
+        }
+
         $commandName = $input->getFirstArgument();
 
+        $trustPath = (new Xdg())->getHomeConfigDir() . '/phpactor/trust.json';
```

Comment:
I believe this should live in XDG DATA not CONFIG directory, ie in `~/.local/share`